flask/app/routes.py

```diff
@@ -517,3 +518,45 @@ def delete_tissue(id: int):
         return "Tissue has dataset(s), cannot delete", 422
     else:
         return "Not Found", 404
+
+
+@app.route("/api/families", methods=["POST"])
+@login_required
+def families():
+    if not request.json:
+        return "Request body must be JSON", 400
+
+    try:
+        updated_by = current_user.user_id
+        created_by = current_user.user_id
+    except:  # LOGIN_DISABLED
+        updated_by = 1
+        created_by = 1
+
+    fam_codename = request.json.get("family_codename")
+
+    # not sure if this is needed or if a different check would be better
+    if len(fam_codename) == 0:
+        return "Family codename must be at least 1 character long", 422
```

Comment:
Empty strings are falsy so you could just do `if not fam_codename` and this should be a 400 Bad Request (missing field) rather than an unprocessable entity.